packages/ListBox/test/cypress/ListBox.a11y.cypress.js

```diff
@@ -0,0 +1,86 @@
+import { getStoryUrlPrefix } from "../../../../.storybook/storyTree";
+import { toggleDropdown } from "../helpers/toggleHelpers";
+import selectors from "../helpers/selectors";
+
+const storyPrefix = `${getStoryUrlPrefix("ListBox")}`;
+
+function arrowUpKeyPress() {
+  cy.focused().type("{uparrow}", { force: true });
+}
+
+function arrowDownKeyPress() {
+  cy.focused().type("{downarrow}", { force: true });
+}
+
+function optionHasFocus(optionName) {
+  cy.get(selectors.popoverList)
+    .findByText(optionName)
+    .should("have.focus");
+}
+
+describe("ListBox MultiSelect a11y", () => {
+  beforeEach(() => {
+    cy.visitStorybook(`${storyPrefix}-backyard-tests--a-11-y-tests-story`);
+    toggleDropdown();
+    cy.wait(350);
```

Comment:
Wait's are needed throughout this test to make sure that the tests are not flaky. This is due to the focus timeouts / popover animation

Reply:
That popover animation :/ still not sure of the benefits of having it lol... like the animation in our panels...